java/clients/src/test/java/sleeper/clients/SleeperClientTest.java

```diff
@@ -125,6 +131,26 @@ void shouldQueryTable() throws Exception {
                 new Record(Map.of("key", "B")));
     }
 
+    @Test
+    void shouldIngestParquetFilesFromS3() {
+
+        String tableName = "";
+        String jobId = UUID.randomUUID().toString();
+        List<String> fileList = List.of("filename1.parquet", "filename2.parquet");
+
+        // When
+        String output = sleeperClient.ingestParquetFilesFromS3(tableName, jobId, fileList);
+
+        // Then
+        assertThat(ingestQueue)
+                .containsExactly(IngestJob.builder()
+                        .tableName(tableName)
+                        .id(jobId)
+                        .files(fileList)
+                        .build());
+        assertThat(jobId).isEqualTo(output);
```

Comment:
The convention with AssertJ is to put the actual value first and the expected value second. We can swap these.